playground/mocks/config/responseConfig.js

```diff
@@ -416,6 +416,20 @@ const windowAuthnLoopbackFailfast = {
   ],
 };
 
+const windowAuthnHttpsLoopback = {
+  '/idp/idx/introspect': [
+    'identify-with-device-probing-https-loopback', // 1 (response order)
+  ],
+  '/idp/idx/authenticators/poll': [
+    'identify-with-device-probing-https-loopback', // 2
+    'identify-with-device-probing-https-loopback', // 3
+    'identify', // 4: as a signal of success
+  ],
+  '/idp/idx/authenticators/poll/cancel': [
+    'authenticator-verification-select-authenticator',
+  ]
+};
+
```

Comment:
Can you explain the purpose of `windowAuthnHttpsLoopback` variable? I don't see it used in any other file.